.husky/post-checkout

```diff
@@ -3,6 +3,11 @@
 
 if git diff-tree -r --name-only --no-commit-id $1 $2 | grep -q package.json
 then
-  git clean -xdf src/**/lazy-* projects/**/lazy-*
-  npm install
+  echo
+  echo --------------------------------------------------------------------------------
+  echo The \'package.json\' contains changes. It could be necessary to run \'npm install\'.
```

Comment:
hmm, instead of deactivating it completely, how about reading an environment property to decide if the update should be run. In our project I'm actually pretty fond of this functionality.